### PR TITLE
Add cached response data for a few more models

### PR DESCRIPTION
**Pull request by jwmatthews**
This updates cached data for several additional models.

Also removes a print statement from mock-client.py to clean up output a little.

### CODE REVIEW

**Review by fabianvf — APPROVED** [review]
Fine with merging all these, though the javaenterprise models are not really giving back usable information with the chat style prompts

File: kai/data/vcr/coolstore/gpt-3.5-turbo/-Users-jmatthews-git-jwmatthews-kai-example-coolstore-src-main-java-com-redhat-coolstore-model-InventoryEntity.java.yaml
Comment (fabianvf) [inline kai/data/vcr/coolstore/gpt-3.5-turbo/-Users-jmatthews-git-jwmatthews-kai-example-coolstore-src-main-java-com-redhat-coolstore-model-InventoryEntity.java.yaml:1]: Actually, these -Users prefixed ones are likely stale from before the prefix removal in the IDE, should remove them from the PR

Comment (jwmatthews) [reply kai/data/vcr/coolstore/gpt-3.5-turbo/-Users-jmatthews-git-jwmatthews-kai-example-coolstore-src-main-java-com-redhat-coolstore-model-InventoryEntity.java.yaml:1]: Thank you for catching that, will remove the User prefixed ones.

**Comment by jwmatthews** [comment]
@fabianvf 

- Removed the 'User' cached data 
- Removed the granite javaentr cached data

Ok to merge?

**Comment by fabianvf** [comment]
yep!

**Pull request merged**